infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationController.kt

```diff
@@ -69,6 +69,18 @@ constructor(
         )
     }
 
+    @PreAuthorize(AUTH_VIEW_LAYOUT_DRAFT)
+    @PostMapping("/{$LAYOUT_BRANCH}/validate-merge-to-main")
```

Comment:
Menee ehkä hifistelyksi näitä miettiä, kun ovat niin helppoja vaihtaa, mutta (restimpiä?) urlivariaatioita olisivat ainakin
```
{$LAYOUT_BRANCH}/validate/merge/main
{$LAYOUT_BRANCH}/validation/merge/main
{$LAYOUT_BRANCH}/merge/validate/main